packages/plugin-core/assets/dendron-ws/tutorial/tutorial.3-links-and-graphs.md

```diff
@@ -0,0 +1,45 @@
+---
+id: hAB2tQHsi0CGuhxz5HkTI
+title: Links and Graphs
+desc: ''
+updated: 1624455055929
+created: 1624333278136
+---
+
+### Links
+
+Dendron supports links between your notes, which can help you organically link your thoughts and build up a powerful knowledge graph.
+
+> 🌱 To create a link, just start typing [[ and Dendron will show you a list of notes in your workspace that you can link to. Try it in the editor pane now.
+
+Example: [[tutorial]]
+
+#### Navigating links
+
+To navigate to the note in the link, you can just click on the link in the preview pane. In the editor pane, you can move the cursor into the link and hit `Ctrl+Enter`.
+
+You can switch back to the previous note by pressing `Ctrl+Tab`
+
+#### Backlinks
+
+Take a look at the backlinks panel on the bottom left section of your sidebar. The backlinks panel shows you all notes with links that point to the current note. This is useful for helping to establish context.
```

Comment:
make sure we are in a note that has backlinks. eg. `use lookup and go to the link you just created with the link`